### DISCUSSION TITLE
Last version wow 3.3.5 Crashed

### DISCUSSION THREAD

**Discussion by Gaffran** [TrinityCore/TrinityCore#29757] [General]
I downloaded the Clean version, did not add any mods or anything else. Enters the game and immediately - crashes! There were no such problems before the latest update. And today I downloaded it and it appeared! Is it just me?

DBErrors: [1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '%u' at line 1
Error while parsing SQL. Core fix required.

//=====================================================
Exception code: C0000420 Произошла ошибка подтверждения.

Assertion message: 
...\TrinityCore3.3.5\src\server\database\Database\MySQLConnection.cpp:610 in MySQLConnection::_HandleMySQLErrno ABORTED.

Fault address:  00007FF6851E0E80 01:0000000000A4FE80 E:\WowServer\WowLichKing\worldserver.exe

**Comment by ghost** [comment]
Difficult to say, but it looks like it is more likely to be your problem, maybe something wrong in how your MySQL server reads your SQL data.

Since we don't know all details of your configuration, it will be mostly guessing what could be wrong (unless one of the TC veterans have seen this kind of problem before and could tell you what it is).

> DBErrors: [1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '%u' at line 1
> Error while parsing SQL. Core fix required.

Maybe something broken in your SQL data?

> Assertion message:
> `...\TrinityCore3.3.5\src\server\database\Database\MySQLConnection.cpp:610 in MySQLConnection::_HandleMySQLErrno ABORTED.`

Or could it be that you don't have the version requirements for MySQL server and/or other packages? (Sorry, I am just guessing here.)

**Comment by Shauren** [comment]
Seeing %u suggests that you are using some very outdated custom patch - that is what causes your crash
